src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,26 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { title, user, body, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <UserInfo user={user} />
```

Comment:
You didn't fix this comment: https://github.com/mate-academy/react_static-list-of-posts-js/pull/149/files#r1282195672

Your app will crash if function `getUserById` return `null`

So, I can suggest next fix:
```suggestion
        user && <UserInfo user={user} />
```

In such way you will check if user is exists otherwise don't display it .